packages/flutter/lib/src/rendering/debug.dart

```diff
@@ -11,7 +10,8 @@ import 'dart:ui' as ui;
 import 'package:flutter/painting.dart';
 import 'package:flutter/rendering.dart';
 import 'package:flutter/scheduler.dart';
-import 'package:vector_math/vector_math_64.dart';
```

Comment:
I don't understand why this wasn't complaining before. The code I removed doesn't refer to it as far as I can tell.